tests/console/docker_image.pm

```diff
@@ -18,26 +18,42 @@ use strict;
 sub run {
     select_console 'root-console';
 
-    check_var('VERSION', 'Tumbleweed') || die 'Only have docker images for Tumbleweed';
+    my $version = get_var('VERSION');
 
-    my $rpm_name   = 'opensuse-tumbleweed-image';
-    my $image_name = 'opensuse/tumbleweed:current';
+    my $image_name;
+    my $image_path;
+
+    if ($version eq 'Tumbleweed') {
+        $image_name = 'opensuse/tumbleweed:current';
+        $image_path = '/usr/share/suse-docker-images/native/*-image*.tar.xz';
+
+        # For Tumbleweed, the image is wrapped inside an RPM
+        zypper_call "in docker opensuse-tumbleweed-image";
+    }
+    else {
```

Comment:
and I would be more explicit here now by looking for Leap and dying with an error message in an else-branch

Reply:
There is no more explicit way AFAIK. Leap can be everything from `15.0`-`42.3`.

Reply:
Well, we do have the function `is_leap` :)

Reply:
Ok, in that case...